pkg/client/image.go

```diff
@@ -144,17 +150,54 @@ func (c *client) ImagePush(ctx context.Context, imageName string, opts *ImagePus
 	return result, nil
 }
 
-func (c *client) ImageDelete(ctx context.Context, imageName string) (*apiv1.Image, error) {
+func (c *client) ImageDelete(ctx context.Context, imageName string, opts *ImageDeleteOptions) (*apiv1.Image, error) {
 	image, err := c.ImageGet(ctx, imageName)
 	if apierrors.IsNotFound(err) {
 		return nil, nil
 	} else if err != nil {
 		return nil, err
 	}
+	if err != nil {
+		return nil, err
+	}
+	if includesTag(imageName) && len(image.Tags) != 1 {
```

Comment:
Playing with some of the rm behavior and here's something that doesnt work that probably should

First, let's assume ive built and tagged an image once without a "tag" identifier, once with:
```
acorn build -t asdf .
acorn tag asdf asdf:v1
```

That gives me output like:
```
$ acorn images
REPOSITORY   TAG       IMAGE-ID
asdf         latest    8666b8a609b1
asdf         v1        8666b8a609b1
```

I think the crux of this issue is going to be the assumed "latest" tag on the image. I cannot "untag" `asdf:latest` so that I am only left with `asdf:v1`.

This one acts like it deleted it, but didnt:
```
$ acorn images rm asdf:latest
asdf:latest
$ acorn images
REPOSITORY   TAG       IMAGE-ID
asdf         latest    8666b8a609b1
asdf         v1        8666b8a609b1
```

This one gives me an error:
```
acorn images rm asdf
Error: deleting asdf: conflict, cannot delete image asdf because it is tagged in multiple images use -f to force
```

Reply:
so the assumption should be that if a user builds just name not tag that the tag should default to latest?

Reply:
@cjellick think I fixed it but have a question on this behavior. If I build w/ no specified tag then tag v1 then remove latest I no longer can remove w/o the tag name as it thinks that no longer exists. You still are able to remove but have to specify the tag. Is this accurate behavior?
```
jsilverio@joshuas-mbp acorn % ./bin/acorn build -t asdf .  
[+] Building 0.6s (5/5) FINISHED                                                                                                                                                     
60466bb98fa109de8f98623489bd0db37d193a336c9293222289196839c643d0
jsilverio@joshuas-mbp acorn % ./bin/acorn tag asdf asdf:v1 
jsilverio@joshuas-mbp acorn % ./bin/acorn images               
REPOSITORY   TAG       IMAGE-ID
asdf         latest    60466bb98fa1
asdf         v1        60466bb98fa1
jsilverio@joshuas-mbp acorn % ./bin/acorn images rm asdf:latest
asdf:latest
jsilverio@joshuas-mbp acorn % ./bin/acorn images rm asdf       
Error: No such image: asdf
jsilverio@joshuas-mbp acorn % ./bin/acorn images rm asdf:v1
asdf:v1
jsilverio@joshuas-mbp acorn % ./bin/acorn images               
REPOSITORY   TAG       IMAGE-ID

```

Reply:
Actually looks like this is similar behavior to previous if I tag as asdf:v1 only and try to image rm asdf it says No such image:asdf

Reply:
Yeah, I think that is ok